src/session.rs

```diff
@@ -671,6 +672,7 @@ impl Session {
             return None;
         }
 
+        // TODO: This is profile specific and needs to be adjusted, what's this 14?
```

Comment:
Blocker for enabling `SRTP_AEAD_AES_128_GCM`. I couldn't figure out what this `14` is